test/test_core.py

```diff
@@ -9361,6 +9361,7 @@ def test_emscripten_atomics(self):
   @node_pthreads
   def test_emscripten_futexes(self):
     self.set_setting('USE_PTHREADS')
+    self.emcc_args += ['-Wno-nonnull']
```

Comment:
Can we add a comment as to why this warning is being disabled?  Is it strictly needed for the test?

Reply:
Yeah, the test is explicitly checking that passing null returns in EINVAL error, which would trigger a diagnostic in the test.